### ISSUE TITLE
`prefer-array-some` should also check for misuse of `Array.findIndex()`

### ISSUE THREAD

**Issue by KurtPreston** [sindresorhus/eslint-plugin-unicorn#1870]
### Description

**prefer-array-some** checks for situations in which `Array.filter(...).length > 0` and `Array.find(...) != null` could be better written as `Array.some(...)`

It would be useful to also check for a similar suboptimal pattern: `Array.findIndex(...) > -1`

**Example:**

```
myArray.findIndex(someCondition) > -1
```

is equivalent to 

```
myArray.some(someCondition)
```

### Fail

```js
[1, 2, 3].findIndex((i) => i % 2) > -1
```

### Pass

```js
[1, 2, 3].some((i) => i % 2)
```

**Comment by sindresorhus** [comment]
Accepted

**Comment by fisker** [comment]
Should also check `.findLastIndex() > -1` too.
